.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @guillejo @kaozdl @MathiasLantean @brunomichetti @nsantos16
+*       @guilleijo @kaozdl @MathiasLantean @brunomichetti @nsantos16
```

Comment:
maybe you can add @JorgeMichelena and @jiss2891

Reply:
Yes, I want to, the thing is that we wouldn't be following the [RS strandard ](https://www.notion.so/rootstrap/Guide-to-assign-reviewers-Code-Owners-245767e45ebb436d83b437dacadc7b17)